compiler/rustc_middle/src/ty/query.rs

```diff
@@ -218,15 +198,6 @@ macro_rules! define_callbacks {
             #[inline(always)]
             pub fn $name(self, key: query_helper_param_ty!($($K)*)) {
                 let key = key.into_query_param();
-                opt_remap_env_constness!([$($modifiers)*][key]);
-
-                let cached = try_get_cached(self.tcx, &self.tcx.query_caches.$name, &key, noop);
```

Comment:
The point of `try_get_cached` was to be inlinable. The rationale was: checking the cache is the fast path, and the slow path can be pushed to the indirect call.
Now that all query return types are `Copy`, we can drop the closure as last parameter as you did. This may decrease IR count a bit.

Reply:
Yeah, I understand that intent, I just want to see the tradeoff of inlining it vs. not, particularly in terms of the effects on our bootstrap time.